recipes/restinio/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class RestinioConan(ConanFile):
+    name = "restinio"
+    license = "BSD-3-CLAUSE"
+    homepage = "https://github.com/Stiffstream/restinio"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "RESTinio is a header-only C++14 library that gives you an embedded HTTP/Websocket server."
+    topics = ("http-server", "websockets", "rest", "tls-support")
+    exports_sources = ["CMakeLists.txt"]
+    settings = "os", "compiler", "build_type", "arch"
```

Comment:
```suggestion
	    settings = ("compiler", )
```

Reply:
Removing all these settings add a warning during when the package is built : `restinio/0.6.8: WARN: CMake generator could not be deduced from settings`.

I know this warning doesn't really matter because CMake will not generate any build files but do you know if there is a way to remove it ?

Reply:
I have never noticed the warning before, I suspect it's not a problem or else the hooks would fail it

Reply:
It looks like it doesn't build without these settings.